crates/cxx-qt-gen/src/generator/cpp/method.rs

```diff
@@ -274,7 +274,12 @@ mod tests {
         let mut type_names = TypeNames::mock();
         type_names.mock_insert("QColor", None, None, None);
 
-        let generated = generate_cpp_methods(&invokables, &qobject_idents, &type_names).unwrap();
+        let generated = generate_cpp_methods(
+            &invokables.iter().map(|method| method).collect(),
```

Comment:
`&invokables.as_ref()` ?

Reply:
Don't think that worked, I can just use iter().collect() though which is slightly cleaner, and the places that only had 1 or 2 methods in tests I have refactored to just declare the methods outside the vec

Reply:
Yeah seems it doesn't and you'd need to implement a load of other traits